docs/source/llms/tracing/index.rst

```diff
@@ -505,6 +520,21 @@ and the :py:meth:`mlflow.client.MlflowClient.delete_trace_tag` method to remove
 FAQ
 ---
 
+Q: Can I disable and re-enable tracing globally?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes. 
+
+There are two fluent APIs that are used for blanket enablement or disablement of the MLflow Tracing feature in order to support 
+users who may not wish to record interactions with their LangChain models for a brief period, or if they have concerns about long-term storage 
```

Comment:
```suggestion
users who may not wish to record interactions with their trace-instrumented models for a brief period, or if they have concerns about long-term storage 
```

nit: As we will adds more integration with other libraries, we may want to avoid explicit mention of `LangChain` unless it is actually applied to LC only.

Reply:
Great point. Removing LangChain references!